lib/Index/Index.cpp

```diff
@@ -1532,7 +1532,7 @@ bool IndexSwiftASTWalker::reportRef(ValueDecl *D, SourceLoc Loc,
   if (!shouldIndex(D, /*IsRef=*/true))
     return true; // keep walking
 
-  if (isa<AbstractFunctionDecl>(D)) {
+  if (isa<AbstractFunctionDecl>(D) || isa<EnumElementDecl>(D)) {
```

Comment:
I assume `call` isn't being added for regular enumerators because of the `isBeingCalled` check. Should we just check if `getParameterList()` is a `nullptr` here though?

Reply:
Since it’s working, I would prefer to not add any additional checks here. I don’t want to make any assumptions whether construction of regular enum cases might also be modeled as a function call in the future.